Dfe.Data.Common.Infrastructure.CognitiveSearch/Tests/Filtering/SearchFilterExpressionBuilderTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Dfe.Data.Common.Infrastructure.CognitiveSearch.Filtering;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.Filtering.FilterExpressions.Factories;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.Filtering.LogicalOperators.Factories;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.Filtering.Options;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.Tests.Filtering.FilterExpressions;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.Tests.Filtering.FilterExpressions.StubBuilders;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.Tests.Filtering.TestDoubles.StubBuilders;
+using Dfe.Data.Common.Infrastructure.CognitiveSearch.Tests.Search.TestDoubles;
+using FluentAssertions;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Dfe.Data.Common.Infrastructure.CognitiveSearch.Tests.Filtering;
+
+public class SearchFilterExpressionBuilderTests
```

Comment:
Only happy path tests

Reply:
Yip, I'll add a ticket to add some red path tests